iroh-gateway/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "iroh-gateway"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+tokio = { version = "1.17.0", features = ["macros", "rt-multi-thread", "process"] }
+axum = "0.5.1"
+clap = { version = "3.1.8", features = ["derive"] }
```

Comment:
Not necessary for this version, but we should have two different parts in the future, one crate that is the binary interface and one that is library only for embedding (this will be needed for all parts)